### ISSUE TITLE
SystemNotifications not showing settings.

### ISSUE THREAD

**Issue by LostKobrakai** [ryancramerdesign/ProcessWire#1056]
I noticed this javascript error a few minutes ago.
```js
var fadeSpeed = Notifications.options.ghostFadeSpeed;
…
if(fadeSpeed.length == 0) interval = 200 * n; // Cannot read property 'length' of null
```
I quickly looked around what could cause it and found the module not showing any settings in the backend and therefore the options passed to js are all `null`. While I can see why it's not working it would also be great if the javascript default options wouldn't be overwritten with `null` which seems kinda counterproductive.

Version is newest dev (2.5.24)

**Comment by ryancramerdesign** [comment]
Not sure I understand how to reproduce this one. System Notifications settings appear to be okay from here. Anything different I need to click on or install?

**Comment by LostKobrakai** [comment]
I deinstalled the module, as it's my module development installation and I implemented something to a module. I just reinstalled it and it works as expected. I remember a refresh having no effect. But still it wouldn't have crashed the whole javascript if the `null` config values wouldn't have overwritten the defaults configs you included in the javascript. So maybe this could be improved independently from what hid the whole module's settings for me.
